src/components/footer/_macro.spec.js

```diff
@@ -478,6 +478,22 @@ describe('macro: footer', () => {
           expect($('.custom-logo').length).toBe(1);
         });
       });
+
+      it('has the additional logo image and link', () => {
+        const $ = cheerio.load(
+          renderComponent('footer', {
+            legal: EXAMPLE_LEGAL_PARAM,
+            crest: true,
+            logo: {
+              logoURL: '#0',
+              logoImage: '<img src="a-logo.svg" class="additional-logo">',
+            },
+          }),
+        );
+
+        expect($('.ons-footer__additional-logo').attr('href')).toBe('#0');
+        expect($('.ons-footer__additional-logo').find('.additional-logo').length).toBe(1);
```

Comment:
Instead of using `.find` you can use css selectors to select the element you are looking for. E.g. `.ons-footer__additional-logo > img` and then instead of finding the length you can do `.attr('src')).toBe('a-second-logo.svg')`. Its a bit more explicit. Then you also dont need the `additional-logo` class.